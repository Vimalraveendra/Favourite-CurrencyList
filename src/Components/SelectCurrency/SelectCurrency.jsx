import React from "react";
import FavouriteCurrency from "../FavouriteCurrency/FavouriteCurrency";
import './SelectCurrency.scss'


const SelectCurrency = ({ favouriteList}) => {
  return (
    <div className="currency-list">
      <ul className="currency-item">
        {favouriteList && favouriteList.map(item => {
          return (
            <FavouriteCurrency
              key={item.id}
              id={item.id}
              currency={item.currency}
              rate={item.rate}
              
            />
          );
        })}
      </ul>
    </div>
  );
};

export default SelectCurrency;