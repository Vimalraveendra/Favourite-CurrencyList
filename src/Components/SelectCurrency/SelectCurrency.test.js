import React from 'react';
import{shallow} from 'enzyme';
import SelectCurrency from './SelectCurrency'
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
const mockStore = configureStore();
const store = mockStore();

const props={}

it("should render Select Currency  component  with default props and redux store", () => {
    let wrapper = shallow(
        <Provider store={store}>
        <SelectCurrency {...props}/>
        </Provider>
        );
    expect(wrapper).toMatchSnapshot();
  });