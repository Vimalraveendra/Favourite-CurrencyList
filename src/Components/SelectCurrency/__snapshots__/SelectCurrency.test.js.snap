// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Select Currency  component  with default props and redux store 1`] = `ShallowWrapper {}`;
